contrib/debian/copyright

```diff
@@ -45,19 +45,19 @@ Files: src/qt/res/icons/add.png,
        src/qt/res/icons/warning.png
```

Comment:
Nit: Previous lines should have their trailing comma removed. I think only white space (new line, ...) is allowed?